pom.xml

```diff
@@ -1952,6 +1973,44 @@
                         <useNativeGit>true</useNativeGit>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.basepom.maven</groupId>
+                    <artifactId>duplicate-finder-maven-plugin</artifactId>
```

Comment:
What it takes to fix issues instead of ignoring them? Can you please list issues that got without this?

Reply:
Here are the issues:
```
[WARNING] Found duplicate (but equal) resources in [com.google.cloud.bigdataoss:gcs-connector:hadoop2-2.2.8:jar:shaded, org.apache.httpcomponents:httpclient:4.5.13]:
[WARNING]   mozilla/public-suffix-list.txt
[WARNING] Found duplicate classes/resources in compile classpath.
[WARNING] Found duplicate (but equal) resources in [com.google.cloud.bigdataoss:gcs-connector:hadoop2-2.2.8:jar:shaded, org.apache.httpcomponents:httpclient:4.5.13]:
[WARNING]   mozilla/public-suffix-list.txt
[WARNING] Found duplicate classes/resources in runtime classpath.
[WARNING] Found duplicate (but equal) resources in [com.google.cloud.bigdataoss:gcs-connector:hadoop2-2.2.8:jar:shaded, org.apache.httpcomponents:httpclient:4.5.13]:
[WARNING]   mozilla/public-suffix-list.txt
[WARNING] Found duplicate classes/resources in test classpath.

```

Reply:
Second warning is from the `trino-faulttolerant-tests` module.

Reply:
Thank you for the update.